MSH/msh_mesh.cpp

```diff
@@ -1506,7 +1506,7 @@ void CFEMesh::GetNODOnPLY(const GEOLIB::Polyline* const ply,
 	}
 
 	// compute nodes (and supporting points) along polyline
-	double search_radius(this->getMinEdgeLength()); // getSearchLength());
+	double search_radius(this->getMinEdgeLength()/2); // getSearchLength());
```

Comment:
Forget to remove ``/2``.

Reply:
I will do that in a different way. Thanks for your help